shadowsocks/udp.go

```diff
@@ -195,15 +174,46 @@ func (s *udpService) Serve(clientConn net.PacketConn) error {
 				}
 				debugUDPAddr(clientAddr, "Got location \"%s\"", clientLocation)
 
+				ip := clientAddr.(*net.UDPAddr).IP
+				var cipher shadowaead.Cipher
+				unpackStart := time.Now()
+				textData, keyID, cipher, err = unpack(ip, textBuf, cipherData, s.ciphers)
```

Comment:
Maybe rename this unpack so it's clearer it's doing the trial decryption?

Reply:
OK, renamed to `findAccessKeyUDP` to match `findAccessKey` in tcp.go.